packages/shared/src/server/repositories/index.ts

```diff
@@ -12,3 +12,4 @@ export * from "./trace-sessions";
 export * from "./scores-utils";
 export * from "./blobStorageLog";
 export * from "./environments";
+export * from "./automation-repository";
```

Comment:
I would challenge that a repository wrapper around prisma statements is worth it. IMO those are simple enough to be used across the board. My understanding was that it's mainly the clickhouse queries that should receive a more "ORM"-like experience using the repositories.

I would vote to remove this abstraction and inline the logic.